less/misc.less

```diff
@@ -53,6 +53,9 @@ input, textarea {
 	visibility: hidden;
 	top: 0;
 	float: left;
+	.selectlist {
+		display: block !important;
```

Comment:
uncomfortable with an `!important` here. is there a place we can do the following
* check if visible
* if hidden 
 * change visibility transparent
 * show
 * measure
 * reset